src/Codeception/Module/Db.php

```diff
@@ -45,6 +45,9 @@
  * * populate: false - whether the the dump should be loaded before the test suite is started
  * * cleanup: false - whether the dump should be reloaded before each test
  * * reconnect: false - whether the module should reconnect to the database before each test
+ * * ssl_key - path to the SSL key
+ * * ssl_cert - path to the SSL certificate
+ * * ssl_ca - path to the SSL certificate authority
```

Comment:
I would like to see something like `(mysql specific option)` at the end of all three lines.